app/views/gobierto_people/people/_person.html.erb

```diff
@@ -16,7 +16,9 @@
     <%= link_to gobierto_people_person_path(person.slug, date_range_params) do %>
       <h2><%= person.name %></h2>
     <% end %>
-    <div class="soft"><%= person.charge %></div>
+    <div class="soft tipsit" title="<%= person.historical_charges.reverse_sorted.join("<br>") %>">
```

Comment:
- This is a N + 1, are you including the historical_charges relation when loading the person?
- could you do a helper or move it to a presenter?

Reply:
Done